### ISSUE TITLE
[BUG] Subsequent permission request for same object shows old message

### ISSUE THREAD

**Issue by sshetenhelm** [yalelibrary/YUL-DC#2886]
**Story**
Users have the ability to re-request a parent object in Blacklight after their first permission request has expired. After a user fills out a new permission request form and presses the "Submit Request" button, the confirmation page displays the message text from the first permission request a user submitted. 

In Management, the new user-submitted text is displayed.

If possible, we would like the request confirmation page in Blacklight to display the most recent message text submitted from a user, even if they have previously requested an object.

**Screenshots**
I recently (07/02) submitted a request with the user message "Testing if Josh can receive a notification to his gmail address." The message successfully went through into Management:

![Image](https://github.com/yalelibrary/YUL-DC/assets/98423494/62b42f34-dc9e-4e70-95b6-4693165ea5d6)

However, since I had previously requested this object, when I got the "confirmation page" in Blacklight, I saw my previously-entered text:

![Image](https://github.com/yalelibrary/YUL-DC/assets/98423494/349a1bb7-157c-4957-b056-9afbf0bb1ac3)

The above text was submitted for Permission Request #80, that was approved and then expired on 06/30.

![Image](https://github.com/yalelibrary/YUL-DC/assets/98423494/e035be18-99c7-49a1-adb6-9452dda1cbd9)

**Acceptance**

- [ ] Permission request submission confirmation page in Blacklight displays most recent "Reason for request" user submitted text

**Comment by jpengst** [comment]
PR: https://github.com/yalelibrary/yul-dc-blacklight/pull/1026

**Comment by K8Sewell** [comment]
Deployed to Demo with this [commit](https://github.com/yalelibrary/yul-dc-blacklight/pull/931/commits/e4c870f8700db648d561255a18610d66dc2dcca9)

**Comment by sshetenhelm** [comment]
@K8Sewell The screenshot is likely because that object is part of a permission set that does not have terms applied. It should not be unique to your user.

**Comment by K8Sewell** [comment]
Ah, yep, that's it.  I did more testing on other objects and was able to request access to parents in other permission sets.  Also, the bug #2890 may be a blip as well.  I visited other records I had expired status on and they behaved as expected - forgot about the viewing permission we set up to allow admin of a permission set view all of the records in that set.  As for this ticket - when I make multiple requests on the same parent the confirmation page displays my most recent reason instead of a prior one.

 

![Image](https://github.com/yalelibrary/YUL-DC/assets/36549923/b74e8231-6bc3-4794-a9e8-7215bac3b5b5)

and when I manually go back to the confirmation page I see my most recent reason still.
